js/test/units/testunitfmt_en_GB.js

```diff
@@ -340,5 +340,209 @@ module.exports.testunitfmt_en_GB = {
         var str = uf.format(m1);
         test.equal(str, "102.338 kilowatt-hours");
         test.done();
-    }
+    },
+    testUnitFormatWithUsagePersonWeight_Short_en_GB: function(test) {
+        test.expect(1);
+        var m1 = MeasurementFactory({
+            unit: "kg",
+            amount: 58
+        });
+
+        var uf = new UnitFmt({
+            usage: "personWeight",
+            measurementSystem: "imperial",
```

Comment:
These tests should not specify the measurement system. The usage should automatically select the right measures.

Reply:
Without giving a specific measurementSystem value, it returns a metic value and I don't know how to automatically select the right measure..couldn't find the code.  how to select the correct cases unit not follow the default?